tsl/src/remote/async.c

```diff
@@ -162,6 +164,16 @@ async_request_send_internal(AsyncRequest *req, int elevel)
 	remote_connection_configure_if_changed(req->conn);
 
 	if (req->stmt_name)
+	{
+		ret = PQsendQueryPrepared(remote_connection_get_pg_conn(req->conn),
```

Comment:
`PQsendQueryPrepared()` didn't exist in our code, so prepared statements were never executed.